BlueprintUI/Sources/Element/ProxyElement.swift

```diff
@@ -15,11 +15,10 @@ public protocol ProxyElement: Element {
 extension ProxyElement {
```

Comment:
Actually, we could go back to how this was if we make `ProxyElement` conform to `ComparableElement` and check our child's `isEquivalent` manually as well.